boosters/node-beginner/tests/03_exercise.test.js

```diff
@@ -8,10 +8,10 @@ const user = {
   lastName: "Mayer",
   id: uuid(),
 };
-describe("test", () => {
+describe("Create User", () => {
   afterAll(teardown);
 
-  it("should", async () => {
+  it("Should create a user", async () => {
```

Comment:
Small change, but we should keep the test case naming consistent (either all start lower case or all start upper case).